File: lds-grupo-7-APP/app/src/main/java/pp/ipp/estg/urbanmarket/retrofit/models/encomendaFeiranteRetrofit/EncomendaFeirante.kt
package pp.ipp.estg.urbanmarket.retrofit.models.encomendaFeiranteRetrofit

import java.util.Date

data class EncomendaFeirante(
    val id: Int,
    val pedidoId: Int,
    val pago: Boolean,
    val data: Date,
    val valorFinal: Float
)

File: lds-grupo-7-SiteBackend/UrbanMarket/UrbanMarket.Models/EncomendaFeirante.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UrbanMarket.Models
{
    public class EncomendaFeirante
    {
        public int Id { get; set; }

        // Relação com a tabela Pedido
        // public Pedido Pedido { get; set; }

        // Chave estrangeira referente ao Pedido
        // public int PedidoId { get; set; }

        public bool Pago { get; set; }

        public DateTime Data { get; set; }

        public float ValorFinal { get; set; }
    }
}
